larray/core/axis.py

```diff
@@ -197,7 +197,7 @@ def labels(self, labels):
                 for i, tick in enumerate(ticks):
```

Comment:
Have you tried removing the _contain_group_ticks(ticks) code?

Reply:
All tests are green but I am unsure this case (i.e. ticks contain one or several groups) is actually tested somewhere.

Reply:
It was, but the tests have either been removed or commented since this is no longer supported AFAIK